### PR TITLE
Feature/social media icons

### LINKED ISSUE #53: Use Social Media Icons
Screenshot 2021-07-09 at 00 30 39

### PR DESCRIPTION
**Pull request by MadhushaPrasad**

### CODE REVIEW

File: components/Event.tsx [hunk 1]
```diff
@@ -69,17 +77,41 @@ const Event = ({
...
-                </a>
+              <div className="text-sm font-medium  flex flex-row flex-wrap justify-center items-center">
+                <p>SHARE: &nbsp;</p>
+                <FacebookShareButton
+                  title={title}
+                  url={'https://sliit-foss-rsvp.web.app/'}
+                  hashtag="#sliit #sliitfoss #fosslk"
+                >
+                  <FacebookIcon lightingColor="white" round={true} size={25}>
+                    {' '}
```
Comment (sithumonline) [inline components/Event.tsx:88]: Rm `{'    '}`

**Review by sithumonline — CHANGES_REQUESTED** [review]

File: components/Event.tsx (hunk 1 above)
Comment (MadhushaPrasad) [reply components/Event.tsx:88]: what is wrong with this? if we use an element without content we can use it like this.

Comment (MadhushaPrasad) [reply components/Event.tsx:88]: please check other files. there also have these elements.

Comment (sithumonline) [reply components/Event.tsx:88]: If it is auto added, remove it otherwise keep it

**Review by sithumonline — APPROVED** [review]

**Pull request merged**